packages/web-components-stencil/src/components/button/button.tsx

```diff
@@ -0,0 +1,113 @@
+import { Component, Element, Event, EventEmitter, h, Listen, Prop } from '@stencil/core';
+import { UtrechtButtonCustomEvent } from '@utrecht/web-component-library-stencil';
+import { attrBoolean } from '../../utils/helpers/attrBoolean';
+
+@Component({
+  tag: 'lux-button',
+  shadow: { delegatesFocus: true },
+  styleUrl: 'button.scss',
+})
+export class Button {
+  @Prop() appearance!: string; // = 'primary-action-button'
+  @Prop() busy?: boolean;
+  @Prop() disabled?: boolean;
+  @Prop() expanded?: boolean | string | 'false' | 'true';
+  @Prop() pressed?: boolean | string | 'false' | 'true' | 'mixed';
+  @Prop({ attribute: 'readonly', reflect: true }) form?: string;
+  @Prop({ attribute: 'formaction', reflect: true }) formAction?: string;
+  @Prop({ attribute: 'formenctype', reflect: true }) formEnctype?: string;
+  @Prop({ attribute: 'formmethod', reflect: true }) formMethod?: string;
+  @Prop({ attribute: 'formnovalidate', reflect: true }) formNoValidate?: boolean;
+  @Prop({ attribute: 'formtarget', reflect: true }) formTarget?: string;
+  @Prop({ attribute: 'popovertarget', reflect: true }) popoverTarget?: string;
+  @Prop({ attribute: 'popovertargetaction', reflect: true }) popoverTargetAction?: string;
+  @Prop() name?: string;
+  @Prop() value?: string;
+  @Prop() type?: string = 'button';
+  @Prop({ reflect: false }) forceState?: 'active' | 'focus' | 'hover';
```

Comment:
```suggestion
  @Prop() forceState?: 'active' | 'focus' | 'hover';
```

`reflect: false` is default gedrag en hoeft niet expliciet genoemd te worden